examples/SEA076_20230906T0852_nrt.cdl

```diff
@@ -0,0 +1,389 @@
+		:metadata_link = "http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html" ;
+		:naming_authority = "Voice of the Ocean Foundation" ;
+		:netcdf_version = "4.0" ;
+		:optics = "{\'calibration_date\': \'2022-04-04\', \'calibration_parameters\': {\'CDOM_DarkCounts\': 50, \'CDOM_SF\': 0.0909, \'Chl_DarkCounts\': 41, \'Chl_SF\': 0.0121, \'SIG_DarkCounts\': 48, \'SIG_SF\': 3.118e-06}, \'factory_calibrated\': \'Yes\', \'long_name\': \'Wetlabs ECO Puck FLBBCD\', \'make\': \'Wetlabs\', \'make_model\': \'Wetlabs FLBBCD\', \'model\': \'FLBBCD\', \'serial\': \'7485\'}" ;
+		:oxygen = "{\'calibration_date\': \'2020-02-05\', \'factory_calibrated\': \'Yes\', \'long_name\': \'JFE Advantech RINKO-FT\', \'make\': \'JFE Advantech\', \'make_model\': \'JFE Advantech AROD_FT\', \'model\': \'AROD-FT\', \'serial\': \'0041\'}" ;
+		:platform = "sub-surface gliders" ;
+		:processing_level = "L1. Quality control flags from IOOS QC QARTOD https://github.com/ioos/ioos_qc Version: 2.1.0 " ;
+		:project = "SAMBA" ;
+		:project_url = "https://voiceoftheocean.org/samba-smart-autonomous-monitoring-of-the-baltic-sea/" ;
+		:publisher_email = "callum.rollo@voiceoftheocean.org" ;
+		:publisher_name = "Callum Rollo" ;
+		:publisher_url = "https://voiceoftheocean.org" ;
+		:references = "created with pyglider https://github.com/c-proof/pyglider" ;
+		:sea_name = "Baltic" ;
+		:source = "Observational data from a profiling glider." ;
+		:standard_name_vocabulary = "CF STandard Name Table v49" ;
+		:summary = " Part of SAMBA continuous monitoring" ;
+		:time_coverage_end = "2023-09-15T07:37:07.016000000" ;
+		:time_coverage_start = "2023-09-06T08:52:59.375000000" ;
+		:title = "OceanGliders example file for SeaExplorer data" ;
+		:total_dives = 64LL ;
+		:transmission_system = "IRIDIUM" ;
+		string :variables = "time", "latitude", "longitude", "nav_state", "heading", "pitch", "roll", "dive_num", "security_level", "declination", "internal_temperature", "internal_pressure", "desired_heading", "ballast_cmd", "ballast_pos", "linear_cmd", "linear_pos", "angular_cmd", "angular_pos", "voltage", "altimeter", "dead_reckoning", "nav_resource", "conductivity", "temperature", "pressure", "salinity", "chlorophyll", "chlorophyll_raw", "cdom", "cdom_raw", "backscatter_scaled", "backscatter_raw", "oxygen_concentration", "temperature_oxygen", "ad2cp_heading", "ad2cp_pitch", "ad2cp_roll", "ad2cp_pressure", "ad2cp_time", "ad2cp_beam1_cell_number1", "ad2cp_beam2_cell_number1", "ad2cp_beam3_cell_number1", "ad2cp_beam4_cell_number1" ;
+		:wmo_id = "6801665" ;
+		:deployment_start = "2023-09-06T08:52:59.375000000" ;
+		:deployment_end = "2023-09-15T07:37:07.016000000" ;
+		:disclaimer = "Data, products and services from VOTO are provided \'as is\' without any warranty as to fitness for a particular purpose." ;
+		:platform_vocabulary = "https://vocab.nerc.ac.uk/collection/L06/current/27/" ;
+		:contributor_email = "callum.rollo@voiceoftheocean.org, louise.biddle@voiceoftheocean.org" ;
```

Comment:
Could `contributor_email` has only two items while `contributor_name` has more? I don't know the answer for that one.